ci/install-verilator.sh

```diff
@@ -10,7 +10,7 @@ fi
 if [ ! -e "$VERILATOR_ROOT/bin/verilator" ]; then
     echo "Installing Verilator"
     rm -f verilator*.tgz
-    wget https://www.veripool.org/ftp/verilator-4.008.tgz
+    wget https://www.veripool.org/ftp/verilator-4.014.tgz
     tar xzf verilator*.tgz
     rm -f verilator*.tgz
     cd verilator*
```

Comment:
FYI this fails if another (e.g. older) version of the `verilator-<ver>` directory is present
```suggestion
    cd verilator-4.014
```

Reply:
Good catch, thanks!